eth_account/_utils/signing.py

```diff
@@ -44,7 +49,9 @@ def sign_transaction_dict(eth_key, transaction_dict):
     return (v, r, s, encoded_transaction)
 
 
-# watch here for updates to signature format: https://github.com/ethereum/EIPs/issues/191
+# watch here for updates to signature format:
+# https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md
+# https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md
```

Comment:
@kclowes you might go subscribe yourself to the relevant discussions for these eips to be sure we catch any updates that happen.